src/main/java/frc/robot/subsystems/drive/DriveIOCTRE.java

```diff
@@ -20,13 +34,26 @@ public class DriveIOCTRE extends SwerveDrivetrain implements DriveIO {
   private Notifier m_simNotifier;
   private double m_lastSimTime;
 
+  // queues for odometry updates from CTRE's thread
+  private final Lock odometryLock = new ReentrantLock();
+  Queue<Rotation2d> gyroYawQueue = new ArrayBlockingQueue<>(20);
```

Comment:
that 20 maybe shoudl go to Constants.java?

Reply:
Changed